TwoSum.java

```diff
@@ -0,0 +1,37 @@
+import sun.lwawt.macosx.CSystemTray;
+
+import java.util.HashMap;
+
+/*
+Time Complexity: O(n)
+Space Complexity: O(n)
```

Comment:
Please specify what `n` indicates.

Reply:
In time complexity n represents length of the array, as we will be iterating over the array,
In space complexity n represents use of HashMap to run the algorithm